migrations/00013_add_task_arn_instance_id_table.sql

```diff
@@ -4,8 +4,8 @@
 CREATE TABLE task_arns_to_instance_info (
     task_arn     TEXT NOT NULL,
     instance_id  TEXT NOT NULL,
-    public_ip    TEXT NOT NULL,
-    private_ip   TEXT NOT NULL,
```

Comment:
You need to do this in a new migration file called something like 00024_make_ip_columns_optional.sql:

```
-- +goose Up
ALTER TABLE task_arns_to_instance_info ALTER COLUMN public_ip DROP NOT NULL;
ALTER TABLE task_arns_to_instance_info ALTER COLUMN private_ip DROP NOT NULL;

-- +goose Down
ALTER TABLE task_arns_to_instance_info ALTER COLUMN public_ip SET NOT NULL;
ALTER TABLE task_arns_to_instance_info ALTER COLUMN private_ip SET NOT NULL;
```